cli/cmd/api.go

```diff
@@ -32,18 +32,17 @@ var apiHealthCmd = &cobra.Command{
 	Long:         "ping the health endpoint of happy api",
 	SilenceUsage: true,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		happyConfig, err := config.GetHappyConfigForCmd(cmd)
+		happyClient, err := makeHappyClient(cmd, sliceName, "", []string{}, false)
```

Comment:
Might be useful to introduce an options arg with default values

Reply:
It looks like happyClient is always created with the same exact arguments: 
```go
makeHappyClient(cmd, sliceName, "", []string{}, false)
```

Reply:
yea I can cut a new PR to refactor this function